client/src/main/com/sinch/sdk/domains/conversation/models/v1/conversation/request/InjectMessageRequestImpl.java

```diff
@@ -0,0 +1,89 @@
+package com.sinch.sdk.domains.conversation.models.v1.conversation.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.sinch.sdk.core.models.OptionalValue;
+import com.sinch.sdk.domains.conversation.models.v1.ChannelIdentity;
+import com.sinch.sdk.domains.conversation.models.v1.ConversationDirection;
+import com.sinch.sdk.domains.conversation.models.v1.ProcessingMode;
+import com.sinch.sdk.domains.conversation.models.v1.conversation.request.InjectMessageRequestBaseImpl.Builder;
+import com.sinch.sdk.domains.conversation.models.v1.messages.AppMessage;
+import com.sinch.sdk.domains.conversation.models.v1.messages.ContactMessage;
+import com.sinch.sdk.domains.conversation.models.v1.messages.ConversationMessageBody;
+import java.time.Instant;
+
+public class InjectMessageRequestImpl extends InjectMessageRequestBaseImpl
+    implements InjectMessageRequest {
+
+  @JsonIgnore
+  public ConversationMessageBody getBody() {
+    return appMessage()
+        .map(f -> (ConversationMessageBody) f)
+        .orElseGet(() -> contactMessage().orElse(null));
+  }
+
+  protected InjectMessageRequestImpl(
+      OptionalValue<AppMessage<?>> appMessage,
+      OptionalValue<ContactMessage<?>> contactMessage,
+      OptionalValue<Instant> acceptTime,
+      OptionalValue<ChannelIdentity> channelIdentity,
+      OptionalValue<String> contactId,
+      OptionalValue<ConversationDirection> direction,
+      OptionalValue<String> conversationId,
+      OptionalValue<Boolean> injected,
```

Comment:
Can you double check this parameter is part of an injectMessage **request**?

Reply:
Good catch/question.
Currently specs claim it is part of schema for request yes.
May be a refinement of specs could be imagine but having to check with product team.
Currently:
- at worst the field will be ignored by server
- at best: specs are right and user can set this field itself.
Question asked to product's team a new PR could created to synch if specs will be updated